app/server/me_api/credit_transfer.py

```diff
@@ -338,6 +339,22 @@ def post(self):
             }
             return make_response(jsonify(response_object)), 400
 
+        except TransferAmountLimitError as e:
+            db.session.commit()
+            response_object = {
+                'message': "Account limit reached",
```

Comment:
Maybe providing the specific limit in the message would be useful? Not strongly opinionated though.